tests/ut/backends/test_memcached.py

```diff
@@ -1,14 +1,21 @@
+from typing import Optional
 from unittest.mock import AsyncMock, patch
 
 import aiomcache
 import pytest
 
-from aiocache.backends.memcached import MemcachedBackend, MemcachedCache
+from aiocache.backends.memcached import MemcachedBackend as _MemcachedBackend, MemcachedCache
 from aiocache.base import BaseCache
 from aiocache.serializers import JsonSerializer
 from ...utils import Keys, ensure_key
 
 
+class MemcachedBackend(_MemcachedBackend):
+    """Define a concrete version of the abstract class, MemcachedBackend"""
+    def build_key(self, key: str, namespace: Optional[str] = None) -> str:
+        return super().build_key(key, namespace)
+
+
```

Comment:
I think it probably makes more sense to just import MemcachedCache?

I suspect they should also be merged into a single class, I really don't see why they are split into 2 classes to start with..

Reply:
> I suspect they should also be merged into a single class, I really don't see why they are split into 2 classes to start with..

You might be right.  I too don't quite see the value in having them separate, but I would want to spend some more time assessing that.

As a quick fix, I'll move `build_key()` to the `...Backend` classes, and submit an issue for combing the pair of derived classes for each backend implementation.

Reply:
> As a quick fix, I'll move `build_key()` to the `...Backend` classes, and submit an issue for combing the pair of derived classes for each backend implementation.

Submitted https://github.com/aio-libs/aiocache/issues/684